File: Koora_Server/Koora_Server/Models/Match.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Koora_Server.Models
{
    public class Match
    {
        public int id { get; set; }
        public DateTime date_Match { get; set; }
        public Team team_Aller { get; set; }
        public Team team_Retour { get; set; }
        public int res_TeamAller { get; set; }
        public int res_TeamRetour { get; set; }
        public string stade { get; set; }
        public IList<Goal> goals{ get; set; }
        public Champions champions { get; set; }
        public IList<Match_TV> match_TVs { get; set; }
        public int etat { get; set; }
        public string type { get; set; }
    }
}

File: Koora-API/src/app/Models/match.ts
import { Champion } from './champion';
import { Goal } from './goal';
import { MatchTV } from './match-tv';
import { Teams } from './teams';

export class Match {
     id: number;
     // tslint:disable-next-line:variable-name
     date_Match: Date;
     // tslint:disable-next-line:variable-name
     team_Aller: Teams;
     // tslint:disable-next-line:variable-name
     team_Retour: Teams;
     // tslint:disable-next-line:variable-name
     res_TeamAller: number;
     // tslint:disable-next-line:variable-name
     res_TeamRetour: number;
     stade: string;
     goals: Goal[];
     champions: Champion;
     // tslint:disable-next-line:variable-name
     match_TVs: MatchTV[];
     etat: number;
     type: string;
}
